src/modules/toolkits/cuda.nix

```diff
@@ -0,0 +1,33 @@
+{ pkgs, lib, config, ... }:
+
+let
+  cfg = config.toolkits.cuda;
+in
+{
+  options.toolkits.cuda = {
+    enable = lib.mkEnableOption "CUDA toolkit";
+
+    package = lib.mkOption {
+      type = lib.types.package;
+      description = "Which package of cuda toolkit to use.";
+      default = pkgs.cudatoolkit;
```

Comment:
Note: this attribute is almost unmaintained, it's better to use the splayed packages